test/integration/fiscalYear.extra.js

```diff
@@ -0,0 +1,144 @@
+/* global expect, agent */
+
+const helpers = require('./helpers');
+
+describe('(/fiscal) Fiscal Year extra operations', () => {
+  const url = '/fiscal';
+  const date = new Date('2017-08-24');
+
+  const fiscalYear2015 = {
+    id : 1,
+    enterprise_id : 1,
+    label : 'Test Fiscal Year 2015',
+  };
+
+  const fiscalYear2017 = {
+    id : 3,
+    enterprise_id : 1,
+    label : 'Test Fiscal Year 2017',
+  };
+
+  const fiscalYear2017MonthsNumber = 12;
+
+  const accounts = [
+    { id : 81, debit : 77, credit : 0 },
+    { id : 83, debit : 0, credit : 77 },
+    { id : 174, debit : 23, credit : 0 },
+    { id : 190, debit : 0, credit : 23 },
+  ];
+
+  const openingBalance = {};
+
+  /**
+   * this test is supposed to returns the fiscal year 2017
+   * from the date 2017-08-24
+   */
+  it(`GET /fiscal/date?date=${flatDate(date)} returns the fiscal year from a given date`, () => {
+    return agent.get(url.concat(`/date?date=${date}`))
+      .then(res => {
+        helpers.api.listed(res, 1);
+        const [fy] = res.body;
+        expect(fy).to.be.an('object');
+        expect(fy).to.have.property('fiscal_year_id', 3);
+        expect(fy).to.have.property('previous_fiscal_year_id', 2);
+        expect(fy).to.have.property('label', 'Test Fiscal Year 2017');
+
+        /**
+         * dates are returned with timezone information
+         * we extract just the date information
+         * FIXME: timezone issue : we have 2017-12-31 in the database
```

Comment:
Looks like this is an old comment that should be removed.  The test below works without a timezone issue 👍